### ISSUE TITLE
Nioh 2 - The Complete Edition (1325200)

### ISSUE THREAD

**Issue by Hielyr** [ValveSoftware/Proton#4593]
# Compatibility Report
- Name of the game with compatibility issues: Nioh 2 - The Complete Edition
- Steam AppID of the game: 1325200

## System Information
- GPU: Vega 64
- Driver/LLVM version: Mesa 20.2.6
- Kernel version: 5.8.0-7630-generic
- Link to full system information report as [Gist](https://gist.github.com/Hielyr/b822852b253bcb7ad5c31334cb46cf6b): https://gist.github.com/Hielyr/b822852b253bcb7ad5c31334cb46cf6b
- Proton version: 5.13-5

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Attempted to launch with Proton 5.13-5, Proton-6.1-GE-1at first. Under Proton 5.13-5, the game launches a window and then immediate closes / crashes most of the time, but will sometimes launch full screen and just behave as below with Proton GE.

Under Proton-6.1-GE-1, the same window will appear, and immediately minimize. If given enough time, there will be some initial menu audio. Attempting to switch to the minimized game window, the window appears to open, but then immediately minimizes again. After switching several time back and forth, the window will sometimes maximize and display a completely black screen. At this point, the mouse cursor changes to the in-game cursor and you can hear menu sounds if clicking, or pressing direction buttons on a game pad.

## Reproduction
Launch the game.

[steam-1325200.zip](https://github.com/ValveSoftware/Proton/files/5934979/steam-1325200.zip)

**Comment by evopls** [comment]
> At this point, the mouse cursor changes to the in-game cursor and you can hear menu sounds if clicking, or pressing direction buttons on a game pad.

Once at this point I was able to get the game playable with the same fixes that worked for the original Nioh:

1). Disable Esync in launch options (PROTON_NO_ESYNC=1)
2). Force Large Address Aware in launch options (PROTON_FORCE_LARGE_ADDRESS_AWARE=1)
3). Create a config file e.g. in your home directory (nioh.conf) with the line: `dxgi.deferSurfaceCreation = True`
4). Add the launch command: DXVK_CONFIG_FILE="/home/username/nioh.conf" %command%

My full launch options look like this:
`PROTON_NO_ESYNC=1 PROTON_FORCE_LARGE_ADDRESS_AWARE=1 DXVK_CONFIG_FILE="/home/username/nioh.conf" %command%`

Performance seems as expected, I have a crash every 1-2h so far. Videos (e.g. the intro cutscene or videos that showcase skills do not work at all), ingame cutscenes do work fine. The broken videos [seem to have been a thing in the original Nioh as well](https://www.protondb.com/app/485510), would be neat if someone maybe has an idea for a fix there.

**Comment by kisak-valve** [comment]
Hello @evopls, it should be noted that `PROTON_FORCE_LARGE_ADDRESS_AWARE` has been enabled by default since Proton 5.0-4 and manually setting that is not needed.

**Comment by GloriousEggroll** [comment]
Making a nioh.conf file within the game's folder with the following:
```
dxgi.deferSurfaceCreation = True
```
and setting launch option:
```
DXVK_CONFIG_FILE="./nioh.conf" %command%
```
Works here. Did not need to disable esync. and as kisak mentioned LAA is already enabled by default. cutscenes are broken even with mfplat patches.

**Comment by Hielyr** [comment]
In reference to GE's comment about just being able to use the conf file with "dxgi.deferSurfaceCreation = True" set, that definitely does the trick and no other launch options appear to need to be set. However, at least on my system, the game is highly unstable and just loves to crash at any given opportunity. 

Most frequently I notice crashes happen at the timing of several events: 

- Swapping weapons and armor. As soon as a weapon or armor item is selected and the button is pressed to equip the item, a crash may occur.
- Loading a level. As soon as a level load is complete and you hit the button to start the level from the Tips / level information loading screen, the game may crash.
- When receiving a Steam chat message, the game is minimised immediately to display the chat window and proceeds to crash 100% of the time.
- Seemingly crashes routinely in intervals of thirty minutes /one hour, but not every time. I've managed to play for up to three hours without a crash a single time, but in most cases the game seems to crash around the one or two hour mark.
- Summoning from Bloody or Benevolent graves. When holding B to summon, the game will sometimes crash when the gauge is full. This has only happened to me with graves from other players, not the default graves built into the game.

Other than that, when exiting the game via the in-game quit option, many times the game fails to fully close and must be killed via either Steam's stop button or a process manager / terminal. Otherwise the game may remain running and is detected by Steam as still being played.

**Comment by CygnusTerminal** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4593#issuecomment-775995130

About the closing issue, i play on Windows until the game is "playable" (crashes addressed and such), and the game would not close unless minimized, if not, it would remain in a black screen. So it may have to do with that

**Comment by 55JinS77** [comment]
Excuse me but even when I make all these changes, Nioh 2 doesnt work for me. Its always stays running, but no screen appears, and neither is minimized. Game is "working" as it is in theory running, but no screen appears, and no sound happens.
My PC is this:
OS: EndeavourOS LinuxKERNEL: 5.10.16-arch1-1CPU: AMD Ryzen 5 1400 Quad-CoreGPU: AMD Radeon RX 560 (POLARIS11, DRM 3.40.0, 5.10.16-arch1-1, LLVM 11.0.1)GPU DRIVER: 4.6 Mesa 20.3.4RAM: 8 GB

I tried both proton 15.3-6, experimental, proton-6.0 GE-1 and proton-6.1 GE-2. But noone of them fixed the problem. I even tried moving all movie files to trashcan, if maybe the intro movie was the problem (As I heard both Nioh games have issues with movies playing) but it didnt work.
Here is my Proton Log

```
======================
Proton: 1612754589 6.1-GE-2
SteamGameId: 1325200
Command: ['/run/media/username/0E320D610E320D61/INSTALATIONS LINUX D/steamapps/common/Nioh2/nioh2.exe']
Options: {'forcelgadd'}
SteamLinuxRuntime: v0.20210114.1-0-g5a1cbb7
pressure-vessel: 0.20210126.0+srt1 scout 0.20210127.0
soldier: 0.20210126.1 soldier 0.20210126.1
======================
ERROR: ld.so: object '/home/username/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
```

This or similar appears regardless of proton version, or if I empty launch options, or place the "DXVK_CONFIG_FILE=path to nioh.conf" launch option.
Anything I missed?

**Comment by CygnusTerminal** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4593#issuecomment-779484407

Hey, could you upload the log file here so we can take a look at it? It's usually located in your home directory, remember to add "PROTON_LOG=1" to your launch options (without the quotations marks). Oh, and pay no mind to the gameoverlayrenderer error, it's always there, as far as i know, it's nothing to worry about.

**Comment by 55JinS77** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4593#issuecomment-779844285

I posted the proton log already...But anyways:
I already fixed the issue I had. Its was because I installed the game in a drive that is not the one the Linux boot is, somehow WINE/Proton couldnt load, so I just reinstalled the game, but this time on my main drive. Its a beginner mistake, sorry for your inconvenience everyone...

**Comment by kisak-valve** [comment]
Hello @55JinS77, what filesystem are you using with `/run/media/username/0E320D610E320D61`? I suspect that it's an NTFS-formatted partition and Proton interacting with NTFS is known to be finicky.

**Comment by Hielyr** [comment]
I agree that it might likely be a formatting issue on the partition. I use two different drives that are not my boot drive for my Steam games and I've never encountered an error like that when I use a Linux FS. I primarily use ext4 at the moment but I have used Btrfs with no issue as well.

**Comment by 55JinS77** [comment]
> Hello @55JinS77, what filesystem are you using with `/run/media/username/0E320D610E320D61`? I suspect that it's an NTFS-formatted partition and Proton interacting with NTFS is known to be finicky.
Its says at details "fuseblk". And yes, the Linux boot drive its a ext4.

**Comment by Hielyr** [comment]
@55JinS77 if the partition is showing as `fuseblk`, then it is indeed most likely formatted as NTFS and would be the cause of the errors you were getting.

**Comment by 55JinS77** [comment]
> In reference to GE's comment about just being able to use the conf file with "dxgi.deferSurfaceCreation = True" set, that definitely does the trick and no other launch options appear to need to be set. However, at least on my system, the game is highly unstable and just loves to crash at any given opportunity.
> 
> Most frequently I notice crashes happen at the timing of several events:
> 
>     * Swapping weapons and armor. As soon as a weapon or armor item is selected and the button is pressed to equip the item, a crash may occur.
> 
>     * Loading a level. As soon as a level load is complete and you hit the button to start the level from the Tips / level information loading screen, the game may crash.
> 
>     * When receiving a Steam chat message, the game is minimised immediately to display the chat window and proceeds to crash 100% of the time.
> 
>     * Seemingly crashes routinely in intervals of thirty minutes /one hour, but not every time. I've managed to play for up to three hours without a crash a single time, but in most cases the game seems to crash around the one or two hour mark.
> 
>     * Summoning from Bloody or Benevolent graves. When holding B to summon, the game will sometimes crash when the gauge is full. This has only happened to me with graves from other players, not the default graves built into the game.
> 
> 
> Other than that, when exiting the game via the in-game quit option, many times the game fails to fully close and must be killed via either Steam's stop button or a process manager / terminal. Otherwise the game may remain running and is detected by Steam as still being played.

Can confirm. Game runs well, but it can crash occasionally when what you said happens. Its relatively a minor issue, but its annoying. At least that is what I got when using Proton-6.1-GE-2. Can anyone confirm if the game improves by using either proton 5.13 or experimental



> @55JinS77 if the partition is showing as `fuseblk`, then it is indeed most likely formatted as NTFS and would be the cause of the errors you were getting.

Indeed, I even formatted that drive not long ago.

**Comment by aliasP** [comment]
Hey,
I tried to play Nioh 2 this weekend, but it didn't work. As described by Hielyr: When starting the game, the window minimizes immediately. If switched to the game window, it sometimes maximizes, only to minimize again. By switching several times, the window will maximize and display a completely black screen. I made all the changes mentioned here, but nothing worked. 

OS: LInux Mint 19.1 Kernel: 4.15.0-135-generic x86_64
CPU: AMD Ryzen 5 2600 six-core	
GPU: Radeon RX 570 series (POLARIS10, DRM 3.23.0, 4.15.0-135-generic, LLVM 11.0.1) 4.6 Mesa 20.3.4
Vulkan instance version: 1.2.162

I tried Proton 15.3-6 and experimental using the Steam Client, but the problem occurred in both. 
Below are the warning/error lines from my Proton log (file also uploaded):
```
ERROR: ld.so: object '/home/username/.steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: Object '/home/username/.steam/ubuntu12_64/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: Object '/home/username/.steam/ubuntu12_64/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: Object '/home/username/.steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
warn: D3D11CoreCreateDevice: adapter is not a DXVK adapter
warn: OpenVR: Initialization of OpenVR failed
warn: OpenXR: Could not determine size of required Vulkan instance extensions
warn: OpenXR: Unable to get required Vulkan device extensions size
```
I'm not sure how to address this error, since it seems to be a Vulkan-specific issue. But the GPU is running Vulkan just fine (tested with vkcube). Also, I am able to run any other of my games flawlessly. It is only Nioh 2, that is causing problems. 

Any suggestions?


[steam-1325200.log.zip](https://github.com/ValveSoftware/Proton/files/6018371/steam-1325200.log.zip)

**Comment by 55JinS77** [comment]
Anyone played the DLC mission "A Song to Calm the Storm"? Because for me, When I enter the Boss room, game stays as a black screen, and never loads. Anyone here had the same issue? Even when I have all NVIDIA settings off (Altough I have all AMD setup), verified the gamefiles, changed Proton version, and even lowered the graphic settings to low, the same issue happens...

**Comment by evopls** [comment]
That's the very first DLC main mission, correct?

If so I had the same issue there a few days ago and couldn't get it to work either (Nvidia setup). I wanted to know if this is the only time when this happened so I ended up pushing past it via booting up an old Win7 install which worked fine after forcing the game to use DX11. All other transitions of the DLCs worked perfectly fine on Linux but this specific one remains broken for me.

What _should_ be playing is this transition: https://www.youtube.com/watch?v=YTIKNValUEk

The first 8 seconds of that clip are in the /steamapps/common/Nioh2/movie folder with all the other .wmv files that are broken on Linux: DEV1_0102A_MV.wmv in this case.

Usually when the game tries to play one of these it gives an "Failure To Play Movie" in the top right corner with the file name and simply skips it. I'm not 100% sure but this might be the only instance where such a .wmv leads into an ingame cutscene, maybe that's what breaks everything.

Is there anything besides the mfplat patches that GE mentioned further up that's worth giving a shot to make the .wmv files playable?

**Comment by 55JinS77** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4593#issuecomment-785784296

It must be that video. Even tough it should not play, and just be skipped automatically; Somehow, it makes the game freeze. I can comfirm it because I Played the game on my Windows boot, and all played well, and going back to Linux, once the first cutscene happened on the Windows boot, then loading on Linux can you fight the boss.

**Comment by 36PopTarts** [comment]
Hi, none of the fixes above worked for me. I am running Linux Mint 20, NVIDIA Geforce 1080 GPU. Game forces itself to minimize on startup and does nothing after the initial musical sting when I use Proton-6.1-GE-2. If I use any of the Valve distributions of Proton, I don't even get a window in my DE, still hear the musical sting, and I have to kill the process. I don't typically have issues running games in general using Wine or DXVK.
[steam-1325200.log.zip](https://github.com/ValveSoftware/Proton/files/6095653/steam-1325200.log.zip)

**Comment by CygnusTerminal** [comment]
Hello again, how can we go about fixing the crashes after 1 hour/2 hours of playing? I want to help debugging, but i surmise a 1 hour long log would be pretty hefty in size.

**Comment by DigitalDevilSummoner** [comment]
> Hello again, how can we go about fixing the crashes after 1 hour/2 hours of playing? I want to help debugging, but i surmise a 1 hour long log would be pretty hefty in size.

lol seems like even playing that long is a bad idea. Out od curiosity I wanted to see how big a log would even be (1.1 gigs btw). By the end of my session this happened: 
![Screenshot_20210321_210852](https://user-images.githubusercontent.com/43498489/111934384-ace02e80-8a97-11eb-8610-e7aa7aa444dd.png)
As I spent more time playing, more stuff seemed to break, this is what I managed to capture before the game refused to display again after I alt+tabbed out to get the screenshot.

**Comment by Hielyr** [comment]
@CygnusTerminal, @DigitalDevilSummoner - ideally you would compress the log before submitting. Text files compress to very small sizes quite readily. 

Use my initial log for example. Its compressed size is ~800K and its uncompressed size is ~83M.

**Comment by CygnusTerminal** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4593#issuecomment-803726177

Yeah, i know, but when i play the game, i usually spend more than a hour, so a timed crash after 1/2 hours seems like a downer for me, i'll just upload the log and hope that it can be fixed. Sounds like OOM to me.

**Comment by DeathTBO** [comment]
I get a very infrequent crash. Only happens when entering a loading screen? It hasn't crashed enough to determine. Proton GE didn't work at all, but 5.13-6 worked great.

I didn't need ``dxgi.deferSurfaceCreation = True``, but it also had no impact performance,

**Comment by CygnusTerminal** [comment]
> I get a very infrequent crash. Only happens when entering a loading screen? It hasn't crashed enough to determine. Proton GE didn't work at all, but 5.13-6 worked great.
> 
> I didn't need `dxgi.deferSurfaceCreation = True`, but it also had no impact performance,

Good to hear, i am still having problems even with the dxgi override. Seems that proton is not loading it.

**Comment by CygnusTerminal** [comment]
Reporting back. Game seems to be working just fine for me. I only have a weird graphic glitch, that does not get solved with the dxvk.conf file. I can see on the logs that it's getting loaded, but it seems to not be working. I'll attach some screenshots. Running mesa-git master.
![20210413082839_1](https://user-images.githubusercontent.com/50408643/114545574-7fb81380-9c32-11eb-85f8-52c9946cad27.jpg)
![20210413082836_1](https://user-images.githubusercontent.com/50408643/114545577-80e94080-9c32-11eb-9cdc-11809cb82432.jpg)
![20210413082832_1](https://user-images.githubusercontent.com/50408643/114545579-8181d700-9c32-11eb-84c8-10f769cc6ebe.jpg)

**Comment by wwmm** [comment]
I do not know if it is happening to other people but after a reboot this game always crashes in the first attempt to run it. And loads fine in the second attempt with a message about a failure to open the intro movie. I have seen this happening many times. This is the proton log from the first attempt that always fails
[steam-1325200.zip](https://github.com/ValveSoftware/Proton/files/6367814/steam-1325200.zip)

Sometimes in the output of `journalctl` I see messages like this:
```
Couldn't set protonvideoconverter60 to READY
winegstreamer: error: decodebin61: Your GStreamer installation is missing a plug-in.
winegstreamer: error: decodebin61: ../src-gst_base/gst/playback/gstdecodebin2.c(4719): gst_decode_bin_expose (): /GstBin:bin61/GstDecodeBin:decodebin61:
no suitable plugins found:
```
I imagine they are related to the tutorial videos that also fail. But I wonder if that particular message can be solved. Is the Gstreamer installation bundled with Proton really missing a plugin that should be there? Or the real message here is that the necessary plugin is not implemented yet?

**Comment by CygnusTerminal** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4593#issuecomment-825894061

AFAIK, Nioh 2 uses ASF formatted videos, i think they are not implemented on Wine's Gstreamer/Proton's Gstreamer. A quick VLC codec check shows that the codec in use is Windows Media Video 9 (wmv3).

**Comment by wwmm** [comment]
For some reason I am having a black screen when trying to load the `uminyudo boss fight` from the `DLC Tengu's Disciple`. There is some background music going on and it stays this way forever. It never loads the next scene so the fight can start. :-(

**Comment by wwmm** [comment]
Proton log that goes from the moment the game is opened to the point where the uminyudo infinity loading happens and I have to force the game to be closed
[steam-1325200.zip](https://github.com/ValveSoftware/Proton/files/6488131/steam-1325200.zip)

**Comment by CygnusTerminal** [comment]
> For some reason I am having a black screen when trying to load the `uminyudo boss fight` from the `DLC Tengu's Disciple`. There is some background music going on and it stays this way forever. It never loads the next scene so the fight can start. :-(

Seems to be an issue with video decoding. As discussed above, it seems the video needs to be played to enable the boss fight, and since wine (and by extension, proton) still can't decode ASF formatted media (guess what game uses it on its cutscenes/media!), well, the game softlocks. I blame japanese devs for using Microsoft's propietary video codecs/formats, instead of just going a more common codec.

**Comment by wwmm** [comment]
>As discussed above, it seems the video needs to be played to enable the boss fight, and since wine (and by extension, proton) still can't decode ASF formatted media (guess what games uses it on its cutscenes/media!), well, the game softlocks.

Some time after writing my comment I realized that this and the issue I saw were the same. The random crashes and the broken videos are annoying but not enough to make me not enjoying the game. But that uminyudo bug has locked me out of content. I was forced to abort the mission and moving on to the next DLC :-(

**Comment by CygnusTerminal** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/4593#issuecomment-841831119

Yeah, until ASF decoding is tackled, we are out of luck. Not even the dreaded `<Workaround removed by moderator>` workaround works here.

**Comment by kisak-valve** [comment]
Hello @CygnusTerminal, the workaround you mentioned is legally problematic and has been removed.

**Comment by Stynern** [comment]
I've been having the same issues (tengu's disciple not working and regular crashes) and was wondering if there was any updates.
For my part I found that the crashes when I play the game at 1440p fullscreen at 60fps (crashes every 20 ~ 30 minutes or so) but a lot less frequent at 2048*1152 windowed at 30fps (crashes every 1.5 ~ 2 hours).
I had the idea of lowering the resolution to avoid crashes because I've been tweaking around settings in my games and found that Total war warhammer 2 was crashing when using a higher resolution and full screen (I use the windows version). So the problem might not be isolated to nioh.
My specs:
CPU: Ryzen 5 5600X
GPU: Radeon RX 580 8G
16G of ram
Proton: 6.3-8

For the tengu's disciple issue I imagine a workaround could be to go through the cutscene on windows and the videos set to not replay. I haven't tried it so I don't know.

**Comment by wwmm** [comment]
One thing that has always caused "delayed crashes" on my computer is using the `soul core fusion` window. It is not an immediate crash. After sometime some operations that usually work fine just crash if I spend too much time the soul core fusion window. I am not sure about how to debug this.

>For the tengu's disciple issue I imagine a workaround could be to go through the cutscene on windows and the videos set to not replay. I haven't tried it so I don't know.

As I do not have Windows I could no try this. I tried to see if something different happened on other difficulties but as expected the game still tried to load the video.

**Comment by wwmm** [comment]
After months without looking at this game's proton logs I've noticed something that seems new
```
err:   D3D11Device::OpenSharedResource: Not implemented
winegstreamer error: oggdemux0: Internal data stream error.
winegstreamer error: oggdemux0: ../src-gst_base/ext/ogg/gstoggdemux.c(4993): gst_ogg_demux_loop (): /GstBin:bin0/GstDecodeBin:decodebin0/GstOggDemux:oggdemux0:
streaming stopped, reason error (-5)
err:   D3D11Device::OpenSharedResource: Not implemented
```
[steam-1325200.zip](https://github.com/ValveSoftware/Proton/files/7823558/steam-1325200.zip)

It seems Nioh also needs the hard to implement `D3D11Device::OpenSharedResource` support. That is sad...

**Comment by kisak-valve** [comment]
Nioh 2 – The Complete Edition 120 Fps mode not working & incorrect gamespeed

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5584.
@DoomedSouls posted on 2022-02-17T10:15:10:

# Compatibility Report
- Name of the game with compatibility issues: Nioh 2 – The Complete Edition
- Steam AppID of the game: 1325200

## System Information
- GPU: AMD RX 6900XT
- Driver/LLVM version: Mesa 21.3.6-1 / LLVM 13.0.1-1
- Kernel version: 5.16.8-xanmod1-1
- Link to full system information report as [Gist](https://gist.github.com/DoomedSouls/8437ba46f3dcd5fbdc6d2afce4da7282)
- Proton version: 7.0-1

## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



## Symptoms 
Selecting 120 fps will not change the frame rate and game speed is half. Selecting afterwards 30 fps will also no change frame rate. Its stuck at 60 fps with incorrect gamespeed.

[steam-1325200.log](https://github.com/ValveSoftware/Proton/files/8087663/steam-1325200.log)

**Comment by wwmm** [comment]
With the latest work on proton experimental it is now possible to start the `uminyudo fight` and go ahead on the `Tengu's Disciple DLC`.  I will now be able to get the spirit guardian I could not get before. It is a good timing considering I have finished what I wanted to do on Elden Ring :smile:

**Comment by wwmm** [comment]
The videos that were broken are still broken. But now there is the usual GStreamer video placeholder in their place.

**Comment by CygnusTerminal** [comment]
> The videos that were broken are still broken. But now there is the usual GStreamer video placeholder in their place.

Try GE-Proton7-10, for me it works 100%.

**Comment by wwmm** [comment]
>Try GE-Proton7-10, for me it works 100%.

I see. Good to know. But I think I will stay on the upstream Proton for now. After more than 700 hours in the game without these videos I do not mind staying without them :smile: The most annoying part was not being able to play a DLC. And that is fixed now :-)

**Comment by DigitalDevilSummoner** [comment]
The move demos still quietly fail, which is honestly not a huge issue, if only the game wouldn't crash so much.

**Comment by neon-grim** [comment]
Game crashes often when visiting the menus like the weapon skill menus. VRR is functional though when used with gamescope and a fps limit is set.

---

Nioh 2 - The Complete Edition (1325200). Game crashes often after visiting skill menus.

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5853.
@Lialeth posted on 2022-05-22T18:56:15:

# Compatibility Report
- Name of the game with compatibility issues: Nioh 2 – The Complete Edition
- Steam AppID of the game: 1325200

## System Information
- GPU: AMD Radeon RX 6900 XT (sienna_cichlid, LLVM 14.0.0, DRM 3.44, 5.17.7-300.fc36.x86_64)
- Driver/LLVM version: Mesa 22.0.3
- Kernel version: 5.17.7-300.fc36.x86_64
- Link to full system information report as Gist: [gistfile1.txt](https://gist.github.com/Lialeth/64d4e2ae612887a7ebac069084cafadd)
- Proton version: Experimental, 7.0-2, GE 7-10, GE 7-18

## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



[Proton log. The files has a lot of lines.](https://drive.google.com/file/d/16bSq-m2r8FBr9By51MAr11psgdALIGpp/view?usp=sharing)

## Symptoms 

-VRR doesnt work natively under x11 (havent tested on wayland since I use gnome).
- Game crashes often when visiting menus such as the skill assignment menu for a specific weapon or Soul core menus.

Currently these are the launch options which I use. They make the game a bit more stable and gamescope allow for VRR to work.

`gamescope -f -b -e -r 110 -- gamemoderun mangohud PROTON_NO_ESYNC=1 PROTON_NO_FSYNC=1 PROTON_LOG=1 ENABLE_VKBASALT=1 %command%`

## Reproduction

Enter a mission and then open the skill assignment menu for a weapon (odachi for example). Change skills on any stances, then leave. The game will crash before leaving the menu or a few minutes afterwards. Rarely the game doesnt crash at all, just repeat the process again.

**Comment by neon-grim** [comment]
I've been trying to look through the massive logs and I've noticed errors like these popping up frequently:
```
err:   D3D11: CreateGeometryShaderWithStreamOutput: Rasterized stream not supported
wine client error:3ec: write: Bad file descriptor
```
Also since the crashes often happens in menus that have embedded videos, I think this might also be of note:
```
4277.481:0140:0144:fixme:dxva2:device_manager_processor_service_GetVideoProcessorDeviceGuids 000000002BBFDB18, 00000000DFF26388, 000000000011EFB8, 000000000011EF60 semi-stub.
1117800 4277.481:0140:0144:fixme:dxva2:device_manager_processor_service_CreateVideoProcessor 000000002BBFDB18, {5a54a0c9-c7ec-4bd9-8ede-f3c75dc4393b}, 00000000DFF26388, 22, 0, 00000000DFF26358.
1117801 winegstreamer warning: videobox1: not negotiated
1117802 winegstreamer warning: videobox1: ../src-gstreamer/libs/gst/base/gstbasetransform.c(1431): gst_base_transform_reconfigure_unlocked (): /GstBin:bin1/GstVideoBox:videobox1:
1117803 not negotiated
1117804 4278.265:0140:0144:fixme:dxva2:device_manager_processor_service_GetVideoProcessorDeviceGuids 000000002BBFDBB8, 00000000DFF26388, 000000000011EFB8, 000000000011EF60 semi-stub.
1117805 4278.265:0140:0144:fixme:dxva2:device_manager_processor_service_CreateVideoProcessor 000000002BBFDBB8, {5a54a0c9-c7ec-4bd9-8ede-f3c75dc4393b}, 00000000DFF26388, 22, 0, 00000000DFF26358.
1117806 winegstreamer warning: videobox2: not negotiated
1117807 winegstreamer warning: videobox2: ../src-gstreamer/libs/gst/base/gstbasetransform.c(1431): gst_base_transform_reconfigure_unlocked (): /GstBin:bin2/GstVideoBox:videobox2:
1117808 not negotiated
1117809 4278.885:0140:0144:fixme:dxva2:device_manager_processor_service_GetVideoProcessorDeviceGuids 000000002BBFDA78, 00000000DFF26388, 000000000011EFB8, 000000000011EF60 semi-stub.
1117810 4278.885:0140:0144:fixme:dxva2:device_manager_processor_service_CreateVideoProcessor 000000002BBFDA78, {5a54a0c9-c7ec-4bd9-8ede-f3c75dc4393b}, 00000000DFF26388, 22, 0, 00000000DFF26358.
1117811 winegstreamer warning: videobox3: not negotiated
1117812 winegstreamer warning: videobox3: ../src-gstreamer/libs/gst/base/gstbasetransform.c(1431): gst_base_transform_reconfigure_unlocked (): /GstBin:bin3/GstVideoBox:videobox3:
1117813 not negotiated
1117814 4279.119:0140:0144:fixme:dxva2:device_manager_processor_service_GetVideoProcessorDeviceGuids 000000002BBFDD98, 00000000DFF26388, 000000000011EFB8, 000000000011EF60 semi-stub.
```

**Comment by CygnusTerminal** [comment]
It's me again, i built an entirely new PC from scratch (6600xt, Ryzen 5 5600G, 16GB 3200MHz RAM, 1TB NVMe) and i am having weird frame pacing issues. FPS are locked to 60, frametime never goes beyond 16.7, but the game slows down regardless, it feels like the FPS go down for a split second. Temperatures on CPU/GPU/SSD are as expected (50, 48, and 39 respectively). 
Link to gist with info: https://gist.github.com/CygnusTerminal/e26e8de5e44710455b044a58da7905c4

**Comment by LZeugirdor** [comment]
Getting some strange audio hitching from the game, currently my monitor is set to 120 but the game is locked to 60. there's also some frameskipping.

**Comment by YamatoHD** [comment]
Been in troubleshooting hell 2 days,
i have an Arch install that shows "video cannot be played" no matter what i do
and fedora that plays them perfectly fine, no matter what i do (works on ANY proton ver)
i have proton log from both, can someone have a look?
Log ends when i quit immediately when the cutscene starts (or doesnt i Arch's case)
[arch_fedora_nioh2_logs.zip](https://github.com/user-attachments/files/18310515/arch_fedora_nioh2_logs.zip)

**Comment by simifor** [comment]
@YamatoHD The video before the main menu, as well as the videos that play before character creation, play fine for me on arch. Looking at your arch logs it appears you have both mesa and amd proprietary drivers installed and both are being loaded. My guess is that this is causing a conflict and preventing videos from playing, I'd try to uninstall the proprietary drivers first and see if videos start working for you on arch.

**Comment by YamatoHD** [comment]
@simifor yeah, i held pro drivers for ShadPS4 back when this was only way to play bloodborne, i uninstalled them and it helped, thx a lot
Strangely i did not encounter any other issues from having both before

**Comment by kisak-valve** [comment]
Nioh 2 – The Complete Edition (1325200) - glitches

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9542.
@yaganoba posted on 2026-03-02T21:30:24:

# Compatibility Report
- Name of the game with compatibility issues: Nioh 2 – The Complete Edition
- Steam AppID of the game: 1325200

## System Information
- GPU: RX 6800 XT
- Video driver version: Mesa 25.3.3
- Kernel version: 6.18.15
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



[steam-1325200.log.gz](https://github.com/user-attachments/files/25695692/steam-1325200.log.gz)

## Symptoms 

Glitch in fire/flames effects in mission "The Beast Born of Smoke and Flames".

## Reproduction

Launch the game through Steam with Proton Experimental.

Video presenting the issue:

https://streamable.com/oeq7ma
https://send.hostnetwork.xyz/download/8b40a986022e5f5b/#7xT6toenoo6Ch_ewFTaghQ

**Comment by simifor** [comment]
@yaganoba this has been fixed on dxvk and should land on proton experimental in a couple of weeks. You can also change your game launch parameter to fix this by using `DXVK_CONFIG="dxvk.zeroMappedMemory = True" %command%`

**Comment by yaganoba** [comment]
> [@yaganoba](https://github.com/yaganoba) this has been fixed on dxvk and should land on proton experimental in a couple of weeks. You can also change your game launch parameter to fix this by using `DXVK_CONFIG="dxvk.zeroMappedMemory = True" %command%`

@simifor 
Thank you very, very much. This launch parameter indeed has solved the issue. Can't wait for the fix to land in stable proton release. Have a great day!

**Comment by bugreport1** [comment]
HDR is broken on Steam Deck OLED in Proton 10 and newer.
The image is "washed-out" and the Deck display stays in SDR mode.
Last working version was 9.0-4
